packages/runner-ct/src/app/RunnerCt.tsx

```diff
@@ -124,47 +118,53 @@ const App: React.FC<AppProps> = observer(
     return (
       <>
         <main className="app-ct">
-          <div
-            className={cs(
-              'specs-list-drawer',
-              {
-                'display-none': state.screenshotting,
-              },
-            )}
-            style={{
-              transform: isSpecsListOpen ? `translateX(0)` : `translateX(-${drawerWidth - 20}px)`,
-            }}
-          >
-            <ResizableBox
-              disabled={!isSpecsListOpen}
-              width={drawerWidth}
-              onIsResizingChange={setIsResizing}
-              onWidthChange={setDrawerWidth}
-              className="specs-list-container"
-              data-cy="specs-list-resize-box"
-              minWidth={200}
-              maxWidth={windowSize.width / 100 * 80} // 80vw
+          {isOpenMode && (
```

Comment:
Do we need a test for this? It would be fairly easy - give a fake config with `isTextTerminal`, assert list is not rendered